source/_patterns/20-molecules/teaser/teaser.scss

```diff
@@ -594,6 +594,12 @@
 
 .teaser--static {
   width: 100%;
+
+  ul > li {
+    &::before {
+      content: none;
```

Comment:
Merci für die Umsicht! Aber lieber nicht (sämtliche Listenelemente für Meteo überschreiben). Wir müssen sonst
<img width="246" alt="bildschirmfoto 2018-10-05 um 12 03 40" src="https://user-images.githubusercontent.com/7767385/46541765-30ecce80-c8bd-11e8-9bcf-d9a05d39e307.png">
vermutlich etliche !important bemühen

Reply:
Nein, dann musst du es eben gar nicht mehr überschreiben. Du willst ja keine roten Punkte haben und das löse ich damit ja für alle Listen innerhalb der static Collection.

Das kommt dir entgegen.

Reply:
nein, ehrlich nicht! wir möchten das pseudo-Element nutzen

Reply:
zB. hier: <img width="734" alt="bildschirmfoto 2018-10-05 um 16 48 12" src="https://user-images.githubusercontent.com/7767385/46542389-93929a00-c8be-11e8-8535-dc155ac6173b.png">
<img width="720" alt="bildschirmfoto 2018-10-05 um 16 48 22" src="https://user-images.githubusercontent.com/7767385/46542397-98574e00-c8be-11e8-81d6-d54d1259f157.png">

Reply:
Das ist alles sehr unglücklich - also vorallem, dass wir die rote Punkte standardmässig für li's haben und das immer wieder überschrieben werden muss!

Was überwiegt denn bei Meteo? Hat es da mehr Orte, wo ihr es effektiv braucht oder mehr Orte, wo ihr die roten Punkte mit content="" überschreibt? 

Zudem lässt sich das ja auch ohne !important überschreiben.

Reply:
Aber lass es uns vor Ort anschauen - es gibt da noch mehr Meinungen. 
Bist du am Dienstag da?

Reply:
es lässt sich entweder mit !important oder mit einer noch längeren Verkettung von Klassen / Elementen überschreiben :/

Reply:
ich bin am Dienstag da

Reply:
Der Plan für die Listenelemente war der folgende:
<img width="401" alt="bildschirmfoto 2018-10-05 um 17 13 31" src="https://user-images.githubusercontent.com/7767385/46543830-297bf400-c8c2-11e8-97ed-022ad9ea0a40.png"> (sämtliche Listenelemente ohne Klasse (im Fliesstext) => `ul > li:not([class])::before` kriegen einen Böbbel) das sollte in onairnav, _layout und Inline angepasst werden, dann könnte man sämtliche `...::before { content: none; }` entfernen.

Reply:
Ja das wäre das Ziel, dass wir das dann überall entfernen könnten...

Dann schauen wir am Di. noch mit Patrick rein und schreiben dann eine Story um das glattzuziehen.